.travis.yml

```diff
@@ -2,20 +2,29 @@ dist: trusty
 install: true
```

Comment:
Travis CLI reformatted the .yml. Other than that the only thing that changed is the secure environment variable, and the deployment, and one new line of error log printing on failure.